Makefile

```diff
@@ -129,6 +129,26 @@ test-coverage:
 	@export VERSION=$(VERSION); bash -x scripts/test_cover.sh
 .PHONY: test-coverage
 
+## txsim-install: Install the tx simulator/fuzzer.
```

Comment:
[nit] proposal to refer to this tool by one name (tx simulator) for consistency with the rest of the help docs in this Makefile

```suggestion
## txsim-install: Install the tx simulator.
```

Reply:
Agreed